craft_parts/packages/apt_cache.py

```diff
@@ -339,6 +365,46 @@ def update(self) -> None:
 
     # pylint: enable=attribute-defined-outside-init
 
+    def fetch_binary(self, package_candidate, destination: Path) -> Path:
```

Comment:
Can we pass the custom progress logger to the real fetch_binary() and drop this?

Reply:
Yes, this was a leftover from a previous attempt to reduce package installation verbosity.